### PR TITLE
lifecycle tags for policy

### PR DESCRIPTION
**Pull request by mystiberry-okta**
@edjohns-okta and @johngronberg-okta I've made the requested lifecycle tag adjustments, and also moved the operation to the top as we did with the other published topics. Please review.

### CODE REVIEW

File: _source/_docs/api/resources/policy.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 title: Policy
-beta: true
 redirect_from: "/docs/getting_started/policy.html"
 ---
 
-> This API is currently in **Early Access (EA)** status.
-
 # Policy API
 
+> The Policy API is an {% api_lifecycle ea %} [release](/docs/api/resources/policy.html).
```
Comment (johngronberg-okta) [inline _source/_docs/api/resources/policy.md]: why does this page link to itself?

File: _source/_docs/api/resources/policy.md [hunk 2]
```diff
@@ -889,9 +890,10 @@ The following conditions may be applied to the rules associated with Okta Sign O
 <a href="#AuthContextConditionObject">AuthContext Condition</a>
 
 
-## Okta MFA Policy
+## MFA Policy
 {: #OktaMFAPolicy }
 
+> The MFA Policy API is a {% api_lifecycle beta %} [release](/docs/api/resources/policy.html).
```
Comment (johngronberg-okta) [inline _source/_docs/api/resources/policy.md]: same question: should this link somewhere else?

File: _source/_docs/api/resources/policy.md [hunk 3]
```diff
@@ -952,6 +954,8 @@ The following conditions may be applied to the rules associated with Okta MFA Po
 ## Password Policy
 {: #GroupPasswordPolicy }
 
+> The Password Policy API is a {% api_lifecycle beta %} [release](/docs/api/resources/policy.html).
```
Comment (johngronberg-okta) [inline _source/_docs/api/resources/policy.md]: same question: should this link somewhere else?

File: _source/_docs/api/resources/policy.md (hunk 1 above)
Comment (mystiberry-okta) [inline _source/_docs/api/resources/policy.md]: doh, it's meant to link to the page explaining releases. will fix.

**Comment by johngronberg-okta** [comment]
LGTM- though I noticed there's an error on line 1135. Should be {: #DelegationObject }

the colon is missing.

**Pull request merged**